src/status_im2/subs/wallet/wallet.cljs

```diff
@@ -81,5 +81,16 @@
                       (string/starts-with? (string/lower-case (:symbol %))
                                            (string/lower-case query)))
                  sorted-tokens)]
-     (println filtered-tokens "3421342342432")
      filtered-tokens)))
+
+(rf/reg-sub
+ :wallet/current-viewing-account-address
+ :<- [:wallet]
+ :-> :current-viewing-account-address)
+
+(rf/reg-sub
+ :wallet/accounts-without-current-viewing-account
+ :<- [:wallet/accounts]
+ :<- [:wallet/current-viewing-account-address]
+ (fn [[accounts current-viewing-account-address]]
+   (filter #(not= (:address %) current-viewing-account-address) accounts)))
```

Comment:
Very small comment, you can take it or not.

Usually in Clojure we prefer using `remove` + predicate, instead of `filter` + `not` + predicate.
i.e.
`(filter #(not= % :something) [,,,])` -> `(remove #(= % :something) [,,,])`

I think that is related to readability and the semanitc meaning: do you want to remove things from a sequence? vs do you want to only take certain things from a sequence?

At the end it;s the same result.

Reply:
TBH, I wasn't aware of the `remove` macro until now. The suggestion you have provided improves the readability of the code. Thanks for it! 😄